examples/bin/node.sh

```diff
@@ -21,6 +21,9 @@ shift
 
 command=$1
 
+#to mask a Count metric/aggregator for datasource 'a', u need to specify 2metrics in the list 'a.count' & 'a.special_count_metric'
+MASKING_OPTS="-DDATASPARK_DO_MASKING=true -DDATASPARK_EXTRAPOLATION_FACTOR=1.35 -DDATASPARK_PRIVACY_THRESHOLD=200 -DDATASPARK_MASKED_METRICS=data.latencyMs,data.uniq,data.special_count_metric,data.sum,data.count,data2.latencyMs,data2.uniq,data2.special_count_metric,data2.sum,data2.count"
```

Comment:
Should we remove unnecessary metrics in DDATASPARK_MASKED_METRICS. (Like latency)